pkg/cmd/flagtypes/addr.go

```diff
@@ -84,7 +85,10 @@ func (a *Addr) Set(value string) error {
 		a.Host = host
 		a.Port = int(portNum)
 	} else {
-		port := a.DefaultPort
+		port := 0
+		if !isURL {
```

Comment:
This means that if I specify `https://hostname` and the defaultPort is `8080`, I will get `https://hostname` instead of `https://hostname:8080`?  Why would I want that to happen?